### ISSUE TITLE
Better handling of multiple columns on microsites

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov_microsites#380]
@msayoung please see examples

https://site3.demo.microsites.localgovdrupal.org/

![Image](https://user-images.githubusercontent.com/708140/233170761-176662d9-7f2a-4f0b-b6b7-860ba9a18401.png)

![Image](https://user-images.githubusercontent.com/708140/233170771-e2a99c7d-76ee-4c84-b6a0-f803708df176.png)

https://site1.demo.microsites.localgovdrupal.org/

![Image](https://user-images.githubusercontent.com/708140/233170870-328f84eb-8e6e-4d3c-a1e4-069f515cde8d.png)

Are there any straightforward fixes here? Thanks

**Comment by msayoung** [comment]
We should - at smallest screen - have the media and text display in one column. however I expect this wont fix the different heights of the image and text in all cases. 

The footer columns should flip to one column at smaller screens I expect. 

@markconroy - any thoughts / suggestions?

**Comment by markconroy** [comment]
I agree @msayoung. It looks like we are missing a media query for when those footer columns kick in.

**Comment by willguv** [comment]
Ignore this

**Comment by markconroy** [comment]
@willguv That PR you linked to only fixes the "Media with Text". I've a second PR for the footer items in Microsites Base. It will be fixed once this - https://github.com/localgovdrupal/localgov_microsites_base/pull/222 - is released.
